autode/hessians.py

```diff
@@ -312,3 +317,200 @@ def frequencies_proj(self) -> List[Frequency]:
         vib_freqs = self._eigenvalues_to_freqs(lambdas)
 
         return trans_rot_freqs + vib_freqs
+
+
+def calculate_numerical_hessian(species, *args, **kwargs) -> None:
+    """
+    Calculate a numerical Hessian by shifting atoms and evaluating the gradient
+    with or without central differences to evaluate the numerical derivatives.
```

Comment:
Add "sets species.hessian" to docstring and that this only works with analytic gradients